services/tenant-ui/frontend/public/forms/reservation.json

```diff
@@ -0,0 +1,52 @@
+{
+  "formDataSchema": {
+    "type": "object",
+    "properties": {
+      "emailAddress": {
+        "type": "string"
+      },
+      "fullName": {
+        "type": "string"
+      },
+      "phoneNumber": {
+        "type": "string"
+      },
+      "tenantName": {
+        "type": "string"
+      },
+      "tenantReason": {
+        "type": "string"
+      }
```

Comment:
I am not sure I love the idea of having the mandatory, non-removable fields (`emailAddress` and `tenantname`) as part of the dynamic payload: it may make sense to have these two in "traditional" input components instead, so that this file can be defined as empty if nothing more needs to be added.

Reply:
Yeah as this part will eventually be user-config-defined, the mandatory fields might not really be appropriate to be defined there...
Or just have it be documentation that you must have those in there and it will fail if you don't 🤷 . We might already have it 422 on the endpoint if you don't supply them.

Reply:
Agreed. This is just the default and simplest config layout. We could definitely bust things apart as we see fit.

Reply:
I would really prefer to have those fields defined (statically) elsewhere to limit the possibility of issues caused by human error or lack of attention. I realize it is not as straightforward as we're now mixing-up dynamic and non-dynamic fields, but I think that the desired "default" outcome would be to have reservations work even if the custom JSON is empty/missing.

Reply:
Done :)